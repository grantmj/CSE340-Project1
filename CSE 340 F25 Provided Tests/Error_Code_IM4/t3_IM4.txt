TASKS
1 2
POLY
    F1 = x - 2;
    F2(Y,Z) = (x + 2)(x+2) + 

    (5 + (Y + V)(Y+V) 

    ) + K H I B^2 Z;
EXECUTE
    a = F1(2);
    OUTPUT a;
INPUTS
    3 5 6
