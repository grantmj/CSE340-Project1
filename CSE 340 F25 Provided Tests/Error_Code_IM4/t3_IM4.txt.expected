Semantic Error Code IM-4: 5 5 7 7 9 9 9 9 
